SPECIFICATION.md

```diff
@@ -184,60 +185,60 @@ JSON example:
 
 ## Badge Payload Specification
```

Comment:
I like the idea of having a BADGE represent only a single dose. It reduces the complexity of the structure and makes small QR codes easier. We should update the spec to explain this, though, so it's not confusing. Up in "Terms and Definitions."